packages/python-name-that-hash/PKGBUILD

```
@@ -0,0 +1,32 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+# This file is based on the AUR package by amadejpapez <amadej.papez@gmail.com>
+
+pkgname=python-name-that-hash
+_pkgname=Name-That-Hash
+pkgver=1.11.0
+pkgrel=6
+pkgdesc="The Modern Hash Identification System."
+arch=("any")
+url="https://github.com/HashPals/Name-That-Hash"
+license=("GPL3")
+depends=("python" "python-click" "python-rich")
+makedepends=("python-installer" "python-poetry-core" "python-wheel")
+source=("https://files.pythonhosted.org/packages/19/7a/731c96302f39610f1ce6742a4e5b997d621b06f5608dc939f5cd48a639a0/name_that_hash-1.11.0-py3-none-any.whl")
```

```suggestion
https://files.pythonhosted.org/packages/py2.py3/${_pkgname::1}/$_pkgname/${_pkgname//-/_}-$pkgver-py2.py3-none-any.whl
```

cf. https://wiki.archlinux.org/title/Python_package_guidelines#Source